### ISSUE TITLE
Feature request: Support Firefox

### ISSUE THREAD

**Issue by YangYangJun** [chathub-dev/chathub#133]
Are you considering chathub's support for Firefox?

**Comment by amjd** [comment]
+1 for this. This is a great extension, would love to use it from Firefox.

**Comment by chenshengzhi** [comment]
+1

**Comment by tshipenchko** [comment]
+1

**Comment by qifei9** [comment]
+1

**Comment by spphinslove** [comment]
来个火狐的吧

**Comment by zhw883** [comment]
+1

**Comment by lolo77777** [comment]
+1

**Comment by xxxxzb** [comment]
+1

**Comment by t3dium** [comment]
yeah this'd be useful, i currently have to keep a 2nd browser open just for chathub

**Comment by panckreous** [comment]
yes please

**Comment by Jeffery-Hebert** [comment]
@wong2 

Is this in the works?

**Comment by maplepy** [comment]
I don't think it is @Jeffery-Hebert

**Comment by wong2** [comment]
Not yet.

**Comment by ceeeeeeeeeeeeeeeeb** [comment]
+1 等个火狐

**Comment by Johndo1234** [comment]
+1

**Comment by loaykhalifa** [comment]
+1

**Comment by Jorgecuenca1** [comment]
+1

**Comment by Jorgecuenca1** [comment]
i need to use this in firefox , how i can use this, or use bard in extension or web in firefox, help please

**Comment by teun95** [comment]
I just want to put this on the agenda again. It would be very useful.

**Comment by Sceptrum** [comment]
I'm always shocked and sad about how Firefox's support in various matters went that low despite being the open-source alternative.
+1

**Comment by priti921** [comment]
![image](https://github.com/chathub-dev/chathub/assets/44541287/bae1bd8a-44b6-4e9c-a2ff-9ebdbb889a28)
You're telling me even Edge gets the extension but Firefox doesn't?

**Comment by wong2** [comment]
Edge extension is totally compatible with Chrome. While Firefox needs extra work.

**Comment by priti921** [comment]
I see. Will it be on Firefox in the future? I would've bought it if it was...

**Comment by maplepy** [comment]
dead project?

**Comment by LouayHanouche** [comment]
Reeeeeally need this for firefox :((

**Comment by maplepy** [comment]
project is dead man

**Comment by LouayHanouche** [comment]
@maplepy  the whole project is dead ?

**Comment by wong2** [comment]
The project is actively maintained, but the code in this repo will no longer be updated because of bad experiences I've had.

Firefox support is not planned at the moment, though you can use the web app in Firefox: https://app.chathub.gg

**Comment by sibbl** [comment]
Still not planned? I'd appreciate to repone the issue.
